src/core/js/views/adaptView.js

```diff
@@ -76,16 +79,26 @@ class AdaptView extends Backbone.View {
    * @param {string} eventName=null Backbone change event name
    */
   changed(eventName = null) {
-    if (!this.isReact) {
+    if (!this.isJSX) {
       throw new Error('Cannot call changed on a non-react view');
     }
     if (typeof eventName === 'string' && eventName.startsWith('bubble')) {
       // Ignore bubbling events as they are outside of this view's scope
       return;
     }
-    const element = render(this.constructor.template.replace('.jsx', ''), this.model, this);
+    const props = {
+      // Add view own properties, bound functions etc
+      ...this,
+      // Add model json data
+      ...this.model.toJSON(),
+      // Add globals
+      ...{
+        _globals: Adapt.course.get('_globals')
+      }
```

Comment:
Did you say this wasn't necessary?

Reply:
Yea, sorry, forgot in all the conversations.